torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3293,6 +3301,15 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    dtypesIfCUDA=None,
                    dtypesIfROCM=None,
                    supports_autograd=False),
+    OpInfo('cdist',
```

Comment:
Should this get a `# TODO: ...` to subsume the cdist tests in test_autograd.py?

https://github.com/pytorch/pytorch/blob/5017c5fcad5889acba1623c634d0aaeadb676aa0/test/test_autograd.py#L7312

Reply:
No, unfortunately, all the tests in test_autograd check different things. Mostly that we get the right output/gradient for boundary values and for very large inputs.
The ones from test_torch that can be moved are moved in the next PR in the stack.